server/aws-lsp-codewhisperer/src/language-server/agenticChat/tools/chatDb/chatDb.ts

```diff
@@ -639,79 +642,123 @@ export class ChatDatabase {
             messages.pop()
             this.#features.logging.debug('Dropped trailing user message')
         }
+
+        //  Make sure there are alternating user and assistant messages
+        let i = 0
+        while (i < messages.length - 1) {
```

Comment:
can we just look at current and previous?

Reply:
yes, will update